src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -82,3 +82,10 @@ export const vaMedicalRecordsIntro = ({ formData }) => {
     <p>Ok, first we’ll ask about your VA medical records related to your {formData.disability.diagnosticText}.</p>
   );
 };
+
+
+export const privateMedicalRecordsIntro = ({ formData }) => {
+  return (
+    <p>Ok, first we’ll ask about your private medical records related to your {formData.disability.diagnosticText}.</p>
```

Comment:
Is this content finalized or is this just temporary text?

Reply:
I don't know that the content is _finalized_ exactly, but it is what was provided in the mockup.
What I'm not sure about is whether we should be pulling from `diagnosticText` or some other property.